changelog.go

```diff
@@ -53,6 +55,8 @@ type Changelog struct {
 	// URLs is a list of links which contain additional information to the
 	// changelog entry such as upstream changelogs or pull requests.
 	URLs []string `json:"urls" yaml:"urls"`
+	// Version is the version of the helm chart.
```

Comment:
```suggestion
	// Version is the Giant Swarm version of the component.
```

I think we should distinguish between the GS version and the upstream version. For now this is just for helm charts but we might use this for other types of components.

Reply:
That is much clear. Thanks.